qhana_plugin_registry/__init__.py

```diff
@@ -148,6 +154,10 @@ def create_app(test_config: Optional[Dict[str, Any]] = None):
 
     api.register_root_api(app)
 
+    # load ui templates from configured files
+    with app.app_context():
```

Comment:
I didn't manage to avoid setting the app context so far. Through the `load_ui_templates` function some methods from `qhana_plugin_registry.db.models.templates` are called, which require the app context (I suppose the initialization isn't finished at this point).

For now, I kept the `with` block but no longer pass the app to the `load_ui_templates` function.

<details>
<summary>Error log without setting app context</summary>

```
Unable to load celery application.
While trying to load the module qhana_plugin_registry.celery_worker:CELERY the following error occurred:
Traceback (most recent call last):
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/.venv/lib/python3.10/site-packages/celery/bin/celery.py", line 57, in convert
    return find_app(value)
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/.venv/lib/python3.10/site-packages/celery/app/utils.py", line 384, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/.venv/lib/python3.10/site-packages/kombu/utils/imports.py", line 56, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/.venv/lib/python3.10/site-packages/celery/utils/imports.py", line 105, in import_from_cwd
    return imp(module, package=package)
  File "/home/user/anaconda3/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/qhana_plugin_registry/celery_worker.py", line 23, in <module>
    create_app()
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/qhana_plugin_registry/__init__.py", line 159, in create_app
    load_ui_templates()
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/qhana_plugin_registry/util/ui_templates.py", line 87, in load_ui_templates
    template_folders = app.config.get("UI_TEMPLATE_PATHS", [])
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/.venv/lib/python3.10/site-packages/werkzeug/local.py", line 316, in __get__
    obj = instance._get_current_object()  # type: ignore[misc]
  File "/mnt/e/Temp/hiwi/qhana-plugin-registry/.venv/lib/python3.10/site-packages/werkzeug/local.py", line 513, in _get_current_object
    raise RuntimeError(unbound_message) from None
RuntimeError: Working outside of application context.

This typically means that you attempted to use functionality that needed
the current application. To solve this, set up an application context
with app.app_context(). See the documentation for more information.
```
</details>